src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/helper/ZipFileHelper.java

```diff
@@ -73,7 +73,7 @@ public static String randomFileName() {
             }
 
             // add metadata
-            zos.putNextEntry(new ZipEntry("metadata.json"));
+            zos.putNextEntry(new ZipEntry("test-data/new-application-payments/metadata.json"));
```

Comment:
??

Reply:
`new ZipEntry("metadata.json")` is not loading a resource, it's just creating an entry with name specified as parameter.

Reply:
Smart intellij